tools/x-storybook/.storybook/webpack.config.js

```diff
@@ -5,7 +5,8 @@ const xEngine = require('@financial-times/x-engine/src/webpack');
 const CopyPlugin = require('copy-webpack-plugin');
 const WritePlugin = require('write-file-webpack-plugin');
 
-const repoBase = path.dirname(findUp.sync('monorepo.json'));
+// TODO: Find a less obtuse heuristic?
+const repoBase = path.dirname(findUp.sync('makefile'));
```

Comment:
if we move storybook to top level this is less of an issue?

Reply:
I think so, hopefully we can dedicate some time to that sort of thing sometime soon 🤞